### PR TITLE
Small improvements for Webservice list

### PR DESCRIPTION
**Pull request by tomas862**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | I added small improvements regarding file naming, missing dockblocks and other stuff regarding code quality. This PR is related with https://github.com/PrestaShop/PrestaShop/pull/10208 comments made by @sarjon.  
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | Page can be reached via admin-dev/index.php/configure/advanced/webservice/ and these changes impacts list only. It should work the same as it was in https://github.com/PrestaShop/PrestaShop/pull/10208 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10311)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/config/services/core/grid.yml
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/config/services/core/grid.yml]: should be prestashop.core.grid.query.webservice_key_query_builder

**Review by mickaelandrieu — APPROVED** [review]

**Review by mickaelandrieu — CHANGES_REQUESTED** [review]
small update of service name expected

**Comment by mickaelandrieu** [comment]
Hello @tomas862,

sorry for the delay, would you mind to rebase so I can add the "waiting for QA" label?

Thanks

**Review by mickaelandrieu — APPROVED** [review]

**Review by mickaelandrieu — DISMISSED** [review]

**Comment by mickaelandrieu** [comment]
Hello, let's validate this by QA.

Once it's ok, I'll build assets again.

Mickaël

**Comment by tomas862** [comment]
Hello @mickaelandrieu  - I can rebase this Pull request this evening, thanks for review :)

**Comment by mickaelandrieu** [comment]
Hello @tomas862, let's wait for QA approval first :)

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Good job @tomas862!

**Pull request merged**
